src/src/WebServer/RootPage.cpp

```diff
@@ -255,6 +255,7 @@ void handle_root() {
       html_end_table();
 
       html_BR();
+      if (Settings.Unit == 0) addFormNote(F("Warning: Unit number is 0, please change it if you want to send it to other units."));
```

Comment:
You should also check to see if the unit nr is actually being used.
When not using C013 (and later ESPEasy_NOW mesh), the unit nr doesn't really matter.
So perhaps also check for UDP port not being 0?

On the other hand, if you end up having issues with p2p, you will maybe not look at this page again.
Perhaps it should be shown on the controller page for C013 too? (and maybe next to the UDP port nr field)

Reply:
Was thinking about it but did not find custom controller settings easily. So rootpage with p2p table was my first idea.

Reply:
Jep... definitely too subtle at this position. I also think it should be on the controller page next to the enable checkmark perhaps. or next to the unitnumber checkmark